apps/src/templates/SchoolNotFound.jsx

```diff
@@ -66,7 +50,7 @@ export default class SchoolNotFound extends Component {
                 type="text"
                 name="school_name_s"
                 value={this.props.schoolName}
-                onChange={this.sendNameToParent}
+                onChange={this.handleChange.bind(this, "schoolName")}
```

Comment:
Totally optional: An ES6y way of writing this would be
```jsx
onChange={(...args) => this.handleChange('schoolName', ...args)}
```
That may be _more_ verbose though.